database/sqlite/sqlite_functions.c

```diff
@@ -1338,3 +1346,36 @@ void sql_build_context_param_list(struct context_param **param_list, RRDHOST *ho
 #endif
     return;
 }
+
+
+#define SQL_CHECK_SHUTDOWN      "select status from agent_log;"
+#define SQL_RECORD_AGENT_START  "insert into agent_log (status, date_created) values ('STARTED',  strftime('%s'));"
+
+int is_restart_after_crash()
```

Comment:
This functions is doing what is expected, and the `CRASH` is sent, I have only one doubt, please, take a look in the commands:

```
bash-5.1# sqlite3 /var/cache/netdata/netdata-meta.db
SQLite version 3.35.3 2021-03-26 12:12:52
Enter ".help" for usage hints.
sqlite> SELECT * from agent_log ;
STARTED|1617284132
sqlite> ^D
bash-5.1# ps aux| grep netdata
root     18559  0.0  0.0   6780  2712 pts/1    S+   13:44   0:00 grep netdata
bash-5.1# netdata
2021-04-01 13:44:52: netdata INFO  : MAIN : SIGNAL: Not enabling reaper
bash-5.1# ps aux| grep netdata
thiago   18567  0.8  2.9 21475259940 238908 ?  Sl   13:44   0:07 netdata
thiago   18571  0.0  0.2 21474972656 21340 ?   Sl   13:44   0:00 /usr/sbin/netdata --special-spawn-server
thiago   18773  0.0  0.0   6992  3736 ?        S    13:44   0:00 bash /usr/libexec/netdata/plugins.d/tc-qos-helper.sh 1
thiago   18799  0.0  0.2 723924 21200 ?        Sl   13:44   0:00 /usr/libexec/netdata/plugins.d/go.d.plugin 1
thiago   18804  0.0  0.0 21474908288 7856 ?    S    13:44   0:00 /usr/libexec/netdata/plugins.d/perf.plugin 1 cycles instructions
thiago   18811  1.3  1.6 21474908908 135460 ?  S    13:44   0:11 /usr/libexec/netdata/plugins.d/apps.plugin 1
root     18813  0.5  0.7 21474967784 63676 ?   Sl   13:44   0:04 /usr/libexec/netdata/plugins.d/ebpf.plugin 1
thiago   18815  0.3  0.3  51732 29040 ?        Sl   13:44   0:02 /usr/bin/python /usr/libexec/netdata/plugins.d/python.d.plugin 1
root     18819  0.0  0.0 21474908340 7668 ?    S    13:44   0:00 /usr/libexec/netdata/plugins.d/nfacct.plugin 1
root     21679  0.0  0.0   6780  2756 pts/1    S+   13:59   0:00 grep netdata
bash-5.1# kill -9 18567
bash-5.1# ps aux| grep netdata
thiago   18804  0.0  0.0 21474908288 7856 ?    S    13:44   0:00 /usr/libexec/netdata/plugins.d/perf.plugin 1 cycles instructions
thiago   18815  0.2  0.3  51732 29044 ?        Sl   13:44   0:02 /usr/bin/python /usr/libexec/netdata/plugins.d/python.d.plugin 1
root     21727  0.0  0.0   6780  2700 pts/1    S+   13:59   0:00 grep netdata
bash-5.1# kill -9 18804 18815
bash-5.1# sqlite3 /var/cache/netdata/netdata-meta.db
SQLite version 3.35.3 2021-03-26 12:12:52
Enter ".help" for usage hints.
sqlite> SELECT * from agent_log ;
STARTED|1617284132
sqlite> ^D
bash-5.1# ps aux| grep netdata
root     21805  0.0  0.0   6780  2748 pts/1    S+   13:59   0:00 grep netdata
bash-5.1# ccnd 2> err >out
bash-5.1# less err 
bash-5.1# netdata
2021-04-01 14:04:34: netdata INFO  : MAIN : SIGNAL: Not enabling reaper
bash-5.1# sqlite3 /var/cache/netdata/netdata-meta.db
SQLite version 3.35.3 2021-03-26 12:12:52
Enter ".help" for usage hints.
sqlite> SELECT * from agent_log ;
STARTED|1617284132
``` 

The `ccnd` is a script that I use to compile Netdata with all flags. And let us ignore the fact that `perf.plugin` and `python.d.plugin` stay alive after the crash, my doubt is: why are we not updating the STARTED? I think it would be good to update it every time we start the agent.

Reply:
You are correct it's not updated every time  -- its not even used at this point. I will go ahead and simplify the PR to use a file at this point since it is not doing something intelligent and it is an overkill.

Reply:
Switched to file